Dockerfile

```
@@ -0,0 +1,26 @@
+# Start with Python base image
+FROM python:3.9-slim
+
+# Set the working directory
+WORKDIR /app
+
+# Copy necessary files
+COPY . /app
+
+# Install dependencies
+RUN pip install wallycore
```

```suggestion
RUN pip install --no-cache-dir wallycore
```
Do not use cache when installing packages to decrease the size of the resulting image.